src/main/java/core/basesyntax/main/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.main;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.suppliers.FigureSupplier;
+
+public class Main {
+    private static final int NUMBER_OF_FIGURES = 6;
+
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[NUMBER_OF_FIGURES];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+            if (i < NUMBER_OF_FIGURES / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+                System.out.println(figures[i].draw());
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+                System.out.println(figures[i].draw());
+            }
```

Comment:
```suggestion
            if (i < NUMBER_OF_FIGURES / 2) {
                figures[i] = figureSupplier.getRandomFigure();
            } else {
                figures[i] = figureSupplier.getDefaultFigure();
            }
            System.out.println(figures[i].draw());
```
avoid code duplication